asker/backend/package.json

```diff
@@ -34,10 +36,10 @@
   "scripts": {
     "lint:check": "eslint --ext '.js,.ts' **/*.ts",
     "lint:fix": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
-    "compile": "npx tsc -b tsconfig.json",
+    "compile": "npx ttypescript -b tsconfig.json",
```

Comment:
typo here? `ttypescript` or it should be `typescript`

Reply:
this is not a typo, is the new tool used!